docroot/themes/humsci/humsci_basic/src/scss/admin/_wysiwyg-editor.scss

```diff
@@ -98,3 +98,16 @@ drupal-media {
 .ck-editor .ck.ck-toolbar.ck-toolbar_grouping > .ck-toolbar__items {
   flex-wrap: wrap;
 }
+
+body.gin--edit-form .views-exposed-form .select-preact {
+  label,
+  select {
+    visibility: visible !important;
+    height: auto !important;
+    position: relative !important;
+  }
+
+  div {
+    display: none;
+  }
```

Comment:
This is not what we need. We still want to use the preact plugin (it has extra features —like search— that can be useful), but it should look like the regular select.